site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -243,7 +243,7 @@ private function editPage(Gradeable $gradeable, $semester, $course, $nav_tab = 0
 
     /* Http request methods (i.e. ajax) */
 
-    private function tryGetGradeable($gradeable_id) {
+    protected function tryGetGradeable(string $gradeable_id, bool $render_json = true) {
```

Comment:
What is this change doing?

Reply:
In AbstractController, trygetGradeable has the protected method header. Giving the function the wrong method header results in an on page exception.